keras_cv/models/object_detection/retinanet/retinanet_test.py

```diff
@@ -236,9 +238,18 @@ def test_saved_model(self, save_format, filename):
 
 @pytest.mark.large
 class RetinaNetSmokeTest(tf.test.TestCase):
-    def test_resnet_backbone_preset(self):
-        for preset in ResNetBackbone.presets:
-            if preset in ResNetBackbone.presets_with_weights:
+    def test_backbone_preset(self):
+        backbones = [
```

Comment:
I think we can/should probably just take all the presets from `keras_cv.models.Backbone`

(Then we don't need to update this when we add new backbones).

I'm still a little hazy on why we need to test this with every variant of every backbone -- @jbischof what's your take there? (Either way, if this is now testing many backbones it should probably be marked extra_large)

Reply:
Sorry, this is not what I meant! I was suggesting to change the `test_backbone_preset` test to use the `presets` field instead of a static list of presets that need to be updated manually. No new tests required.